src/gtc/numpy/npir_codegen.py

```diff
@@ -420,8 +420,14 @@ def run({{ signature }}):
                 {% for name in field_params %}{{ name }}_ = ShimmedView({{ name }}, _origin_["{{ name }}"])
                 {% endfor %}# -- end data views --
 
+                {% if ignore_np_errstate %}
+                with np.errstate(divide='ignore', over='ignore', under='ignore', invalid='ignore'):
+                {% else %}
+                with np.errstate():
+                {% endif %}
+
                 {% for pass in vertical_passes %}
-                {{ pass | indent(4) }}
+                {{ pass | indent(8) }}
```

Comment:
```suggestion
                {{ pass | indent(8) }}
                {% else %}
                    pass
```

Reply:
The other two failing tests errors are likely caused by the missing `ignore_err_state` kwarg in the `NpirCodegen.apply()` method. I guess the signature should be updated to reflect that this kwarg is compulsory (although this would likely break compatibility with the superclass for mypy)